lib/Onnxifi/Base.cpp

```diff
@@ -53,11 +53,11 @@ onnxStatus BackendId::checkGraphCompatibility(const void *onnxModel,
     return ONNXIFI_STATUS_INTERNAL_ERROR;
   }
 
-  glow::lower(function, /* loweredMap */ nullptr, glowBackend_.get());
+  CompilationContext cctx;
+  glow::lower(function, cctx, glowBackend_.get());
 
   // Call the backend's transformPostLowering to match the normal compilation
   // pipeline then DCE any nodes that are no longer needed.
-  CompilationContext cctx;
   cctx.compMode = CompilationMode::Infer;
```

Comment:
I would move this line setting the compilation mode up to where you moved creating the CompilationContext to. I don't think it's valid to have a context without the mode set.

Reply:
done. Btw, i found that by default CompilationContext set its mode as Infer, so probably we can also remove this line here